tools/validators/instance_validator/instance_parser.py

```diff
@@ -81,14 +82,13 @@ def _load_yaml_with_schema(filepath, schema):
   try:
     parsed = syaml.load(content, schema)
 
-    return parsed
+    return (parsed, None)
   except (ruamel.yaml.parser.ParserError,
           ruamel.yaml.scanner.ScannerError,
           syaml.exceptions.YAMLValidationError,
           syaml.exceptions.DuplicateKeysDisallowed,
           syaml.exceptions.InconsistentIndentationDisallowed) as e:
-    print(e)
-    return None
+    return (None, e)
```

Comment:
`exception`?

Reply:
returning the exception is enough why the need to return None as well?

Reply:
Because the function is now defined to return a tuple